cmd/clusterctl/examples/openstack/provider-component/user-data/centos/templates/master-user-data.sh

```diff
@@ -14,6 +14,7 @@ POD_CIDR={{ .PodCIDR }}
 SERVICE_CIDR={{ .ServiceCIDR }}
 ARCH=amd64
 
+swapoff -a
```

Comment:
Hmm... is this enough in this context? If there is swap enabled in the image, is there a need to remove id from fstab as well?

Reply:
maybe ok because kubeadm has some dependency? this user-data will only be executed once ...

root@kvm-017751:~/go/src/sigs.k8s.io/cluster-api/cmd# swapon -a
root@kvm-017751:~/go/src/sigs.k8s.io/cluster-api/cmd# kubectl get nodes
NAME       STATUS   ROLES    AGE    VERSION
minikube   Ready    master   120m   v1.12.7

Reply:
yeah. But they check it because having swap enabled is a bad thing for kubernetes, and after rebooting, the swap will be enabled again (if there are things left in fstab).